Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -1086,17 +1086,21 @@ private void OnInternalOwnerEscaping(EscapingEventArgs ev)
                     continue;
 
                 ev.Player.RemoveItem(item);
-
                 TrackedSerials.Remove(item.Serial);
-
-                Timing.CallDelayed(1.5f, () => Spawn(ev.Player.Position, item, null));
+                ev.Player.SessionVariables.Add("CustomItem", item);
```

Comment:
This is a novel idea, but will entirely break if the player has multiple custom items.

Reply:
true true i will change it in a better way "CustomItem-{Serial]"